src/renderer/views/Watch/Watch.js

```diff
@@ -591,7 +591,7 @@ export default Vue.extend({
             channelName: result.author,
             channelId: result.authorId
           })
-
+          console.log('recommended:', result.recommendedVideos)
```

Comment:
Should we include these changes in the app...?
Not just this PR but all `console.log`

Reply:
I would be okay with getting rid of console.log. i also think some errors in the app currently use "console.log" instead of "console.error" and it would probably be best if those were changed

Reply:
Could maybe be  enforced by the linter in the future...